frontend/dockerfile/docs/reference.md

```diff
@@ -2375,10 +2728,94 @@ environment variable expansion semantics could be modified.
 The `SHELL` instruction can also be used on Linux should an alternate shell be
 required such as `zsh`, `csh`, `tcsh` and others.
 
+## Here-Documents
+
+> **Note**
+>
+> Added in [`docker/dockerfile:1.4`](#syntax)
+
+Here-documents allow redirection of subsequent Dockerfile lines to the input of
+`RUN` or `COPY` commands. If such command contains a [here-document](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_04)
+Dockerfile will consider the next lines until the line only containing a
```

Comment:
```suggestion
the Dockerfile will consider the next lines until the line only containing a
```

Reply:
Generally, avoid future tense in tech-docs; e.g. this could be

```markdown
the Dockerfile considers ...
```